data/stats/get_runway_stats.py

```diff
@@ -0,0 +1,39 @@
+"""
+    Gather runway statistics for each 
+    airport from given airport runway data. 
+"""
+
+from dataclasses import dataclass, field
+import pandas as pd
+from src.data.types import Runway
+
+@dataclass
+class RunwayStats:
+    runways: pd.DataFrame
+    count: pd.DataFrame = field(init=False)
+    len: pd.DataFrame = field(init = False)
+
+    def __post_init__(self) -> None:
+        self.count = pd.DataFrame(self.runways.map(self.count_runways))
+        self.len = pd.DataFrame(self.runways.map(self.sum_runways_len))
```

Comment:
I'd be inclined to use more intuitive names for these, something like `runway_count` and `total_runway_len`?